tests/e2e/upgrades_test.go

```diff
@@ -221,7 +221,7 @@ var _ = Describe("ManagedOSImage Upgrade e2e tests", func() {
 		})
 
 		It("applies an os-upgrade from a channel", func() {
-			mr := catalog.NewManagedOSVersionChannel(
+			mr := catalog.LegacyNewManagedOSVersionChannel(
```

Comment:
Using the legacy method for now, this should be refactored in the context of #240